ansible/playbooks/AdoptOpenJDK_Unix_Playbook/trss.yml

```diff
@@ -116,10 +116,10 @@
             owner: "{{ Jenkins_Username }}"
             group: "{{ Jenkins_Username }}"
 
-        - name: npm ci && npm run build of test-result-summary-client
+        - name: Npm ci && npm run build of test-result-summary-client
```

Comment:
As before `npm` is the command so should not be switched to `Npm`. Perhaps prefix with `Run:` instead?
Same for the other occurrence(s) later